### ISSUE TITLE
Cannot run the demo code

### ISSUE THREAD

**Issue by threefoldo** [FeliMe/AudioStyleNet#2]
First, I compiled the 'fused' op and install the missing face_alignment:
```
  cd op && python setup.py install
  pip install face_alignment

```
Then, this error occurred:

```
FileNotFoundError: [Errno 2] No such file or directory: 'model/audio2expression_convNet_justus.pt'
```

This model file is missing in google drive.

**Comment by sharvil** [comment]
Yeah, I also ran into the same issues; currently blocked on missing pretrained model.

**Comment by Johnson-yue** [comment]
where is the paper？ how to train it？

**Comment by FeliMe** [comment]
> First, I compiled the 'fused' op and install the missing face_alignment:
> 
> ```
>   cd op && python setup.py install
>   pip install face_alignment
> ```
> 
> Then, this error occurred:
> 
> ```
> FileNotFoundError: [Errno 2] No such file or directory: 'model/audio2expression_convNet_justus.pt'
> ```
> 
> This model file is missing in google drive.

Unfortunately, I am not allowed to provide this model as it is from another work. You would have to train the model from scratch. Sorry about that.

**Comment by FeliMe** [comment]
> where is the paper？ how to train it？

I added the link to the thesis in README.md. Be aware that it is a Master's thesis and there is no other paper for this work.
